cypress/integration/Page/publicAuctionSortPage.js

```diff
@@ -97,21 +58,13 @@ describe('Testcase for Manheim arrangment', () => {
                         arr.push(Number(year))
                     })
                 }).then(checkIncrese => {
```

Comment:
`checkIncrease` instead of `checkIncrese`. Please check and update the code in other places.